resources/language/resource.language.fr_fr/strings.po

```diff
@@ -418,7 +418,7 @@ msgstr "Netflix a renvoyé l'erreur suivante : "
 
 msgctxt "#30103"
 msgid "Error details have been written to the Kodi log file. If the error message does not provide guidance on how to resolve the issue or you continue to receive this error, please report it on GitHub by following the Readme instructions."
-msgstr "Les détails de l'erreur ont été écrits dans le fichier journal de Kodi. Si le message d'erreur ne vous indique pas comment résoudre le problème ou si vous continuez à recevoir cette erreur, signalez-le via GitHub. Veuillez fournir un journal de débogage complet avec votre rapport d'erreur (activez 'Journalisation du débogage' dans les paramètres Kodi)."
+msgstr "Les détails de l'erreur ont été écrits dans le fichier journal de Kodi. Si le message de l'erreur ne vous indique pas comment résoudre le problème ou si vous continuez à recevoir cette erreur, signalez-le sur GitHub en suivant les instructions du Readme"
```

Comment:
missing final dot

Reply:
Nice catch :) I will fix it in my next commit.